### DISCUSSION TITLE
Anytone D878UV is not detected by qdmr

### DISCUSSION THREAD

**Discussion by DaniAngel** [hmatuschek/qdmr#438] [General]
I just installed qdmr on linux mint 20.3.  When I connect the radio, Anytone D878UV, via the usb cable adapter, I cannot get qdmr to detect it.  When I click detect, it gives me an error that no matching device was found.  I checked my groups permission with the groups command in the console and dialout is listed.  Is there something else I should try?

**Comment by galagithub** [comment]
can you post a "lsusb" output with the cable inserted and another "lsusb" output with the cable removed from the PC (complete disconnect).

sudo might be required to use lsusb.

**Comment by allesand** [comment]
@DaniAngel : is it a 878UV (as in the version with 200k memory entries, the older one) with the supplied "passive" cable? Both my 868/878UV (upgraded firmware) and my 878UVII work as expected with the cable that came with the 878UVII, on three different "firmware trains". But yes, "lsusb" for the VID:PID would be interesting. Also, if you start QDMR in a terminal, that output might be helpful. How old is you radio? A few weeks ago a new 878 UVII Plus worked as expected.
Did you install QDMR via snap? If so, please see 
https://dm3mat.darc.de/qdmr/install.html#snap
If it is a new radio: does the battery already have USB-C-charging? (Trying to determine whether s.th. changed in recent models)

**Comment by galagithub** [comment]
if the cable style is audio jack like, then that is NOT a passive cable, but a USB to serial converter.

when taking the "lsusb" output with no cable attached, make sure the cable is completely removed/unplugged from the PC.

**Comment by allesand** [comment]
The Anytone radios use a passive cable, as does the Baofeng RD-5R (though with different pin layout). The Retevis RT3 cable is an active one, at least judging by the size of the plug.

**Comment by hmatuschek** [comment]
I do not believe, that is a direct issue with qdmr. Move that to the discussions.
